rules/no-implicit-start.js

```diff
@@ -29,6 +29,10 @@ module.exports = function() {
       return false;
     }
 
+    if (is(node, 'bpmn:Activity') && is(node.$parent, 'bpmn:AdHocSubProcess')) {
+      return false;
+    }
+
     if (is(node, 'bpmn:SubProcess') && node.triggeredByEvent) {
       return false;
```

Comment:
Must not be false, I think, cf. https://github.com/bpmn-io/bpmnlint/pull/181#issuecomment-2739396363